### PR TITLE
docs: Legal Assistant name change

### PR DESCRIPTION
**Pull request by aldunson**
changing name from legal assistant to Contract Review and Summarization assistant

## Purpose

* ...

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by adamdougal** [comment]
@aldunson Heya! Can you explain the motivation for this change? Also, please see license agreement.

**Comment by aldunson** [comment]
Hi @adamdougal, these changes are being made after a decision was made to broaden the use case for the 'legal assistant' accelerator that build on CWYD. The decision was approved by leadership and stakeholders.

**Comment by aldunson** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: docs/TRANSPARENCY_FAQ.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Chat with your data Solution Accelerator: Responsible AI FAQ
```
Comment (adamdougal) [inline docs/TRANSPARENCY_FAQ.md:1]: Can we lowercase the filename? 

Also wondering if `responsible_ai_faq.md` would be a better name?

Comment (aldunson) [reply docs/TRANSPARENCY_FAQ.md:1]: we have to leave the name as is because it is a CELA approval, but I can make it lowercase.

**Review by adamdougal — APPROVED** [review]
LGTM!

**Pull request merged**